Sources/SwiftProtobuf/BinaryEncoder.swift

```diff
@@ -125,12 +127,24 @@ internal struct BinaryEncoder {
 
     // Write a string field, including the leading index/tag value.
     mutating func putStringValue(value: String) {
```

Comment:
Minor:  I would `let utf8 = value.utf8` to factor out multiple references to the `utf8` view below.